common.sh

```diff
@@ -397,6 +397,13 @@ fi
 
 export ENABLE_LOCAL_REGISTRY=${ENABLE_LOCAL_REGISTRY:-}
 
+# Helper variable, not meant to be configurable by user, will be set if two node fencing is detected
+# and used as reference for TNF configurations.
+export ENABLE_TWO_NODE_FENCING=""
+if [[ ${NUM_ARBITERS} -eq 0 ]] && [[ ${NUM_MASTERS} -eq 2 ]]; then
+  export ENABLE_TWO_NODE_FENCING="true"
```

Comment:
This is a little bit confusing for me. Below there is already a logic to enable the 2n fendince when the AGENT_E2E_TEST_SCENARIO is TNF, which makes sense to me.

Here I was expecting something more like
```
export ENABLE_TWO_NODE_FENCING=${ENABLE_TWO_NODE_FENCING:"false"}
```

Either to avoid an unset var either to allow the user to override it _if_ required (usually when not using AGENT_E2E_TEST_SCENARIO).

Reply:
Ah sorry, I had forgotten another place I wanted to use this, I replaced the `if` logic in the ipi flow as well, so this logic just sets the correct var for that flow. As well we explicitly set the var in the agent flow. 

I updated the code to allow the user to change it if in the future we want to force fencing credentials being present, but otherwise this will just be on auto pilot

Reply:
Oh good catch, updated the code to also check for that so it only gets checked when AGENT_E2E_TEST_SCENARIO is **not** set, and allows the agent flow to dictate that themselves, thanks @andfasano !